src/main/java/com/gmail/shablinski93/App.java

```diff
@@ -1,117 +1,14 @@
-                    break;
-                case "4":
-                    System.out.println("Введите наименование блюда: ");
-                    String name = reader.readLine();
-                    System.out.println(DishServiceImpl.getInstance().findDishByName(name));
-                    break;
-                case "5":
-                    DishServiceImpl.getInstance().saveInFile();
-                case "6":
-                    break;
-            }
-        }
-
-//        Scanner scanner = new Scanner(System.in);
-//
-//        Connection connection = DriverManager.getConnection(DB_URL, DB_USERNAME, DB_PASSWORD);
-//
-//        while (true) {
-//            System.out.println("1.Показать список блюдов");
-//            System.out.println("2.Добавить блюдо");
-//            System.out.println("3.Посмотреть блюдо по айди");
-//            System.out.println("4.Выход из прилаги");
-//
-//            int command = scanner.nextInt();
-//
-//            if (command == 1) {
-//                Statement statement = connection.createStatement();
-//                String sqlSelect = "select * from dish order by id desc";
-//                ResultSet resultSet = statement.executeQuery(sqlSelect);
-//                while (resultSet.next()) {
-//                    System.out.println("id: " + resultSet.getInt("id") + " dish name:" +
-//                            resultSet.getString("dishname") + " calories: " +
-//                            resultSet.getString("caloriescount") +
-//                            " ingredients: " + resultSet.getString("ingredientcount"));
-//                }
-//            } else if (command == 2) {
-//                Statement statement = connection.createStatement();
-//                String sqlInsert = "insert into dish (id, dishname, caloriescount, ingredientcount) values (?, ?, ?, ?)";
-//                PreparedStatement preparedStatement = connection.prepareStatement(sqlInsert);
-//
-//                System.out.println("enter ID:");
-//                int dishId = scanner.nextInt();
-//                preparedStatement.setInt(1, dishId);
-//
-//                System.out.println("enter dish name:");
-//                String dishName = scanner.next();
-//                preparedStatement.setString(2, dishName);
-//
-//                System.out.println("enter calories count:");
-//                String caloriesCount = scanner.next();
-//                preparedStatement.setString(3, caloriesCount);
-//
-//                System.out.println("enter ingredients:");
-//                scanner.nextLine();
-//                String ingredients = scanner.nextLine();
-//                preparedStatement.setString(4, ingredients);
-//
-//                preparedStatement.executeUpdate();
-//
-//            } else if (command == 4) {
-//                System.exit(0);
-//            }
-//        }
+        TaskService taskService = new TaskServiceImpl();
```

Comment:
Just a small naming notice.
Generally it is not Service but Application. So, `TaskApp` sounds better in this case.